devsetup/tripleo/ceph.sh

```diff
@@ -62,3 +62,12 @@ openstack overcloud ceph deploy \
     --network-data network_data.yaml \
     --cephadm-default-container \
     --output deployed_ceph.yaml
+
+# TODO: jgilaber workaround to have ceph working with only 1 compute node
+# should remove it once we move to have 3 computes
+ssh -o StrictHostKeyChecking=no root@192.168.122.103 cephadm shell -- ceph osd pool create images --size 1
```

Comment:
-2: This is a workaround that shouldn't be tied to `install_yamls`.
I'll let @fultonj comment but as per [1] you can pass a `ceph.conf` that will be assimilated during the bootstrap of the cluster, and it will set the option for you.
As described in [2], you can put in an `initial-ceph.conf` something like:

```
[global]
osd_pool_default_size = 1
```
and, as described for the network [3], you can pass it to the `ceph deploy` command (`--config ceph.conf`):

```
openstack overcloud ceph deploy \
  --config initial-ceph.conf ....
```

Let's try this approach that seems more generic, useful for the long term, and it's not a workaround.
@fultonj can you check my comment and see if is accurate?

@katarimanojk FYI

[1] https://github.com/openstack-archive/tripleo-ansible/blob/stable/wallaby/tripleo_ansible/roles/tripleo_cephadm/tasks/bootstrap.yaml#L37C15-L37C46
[2] https://docs.ceph.com/en/latest/install/manual-deployment/
[3] https://docs.openstack.org/project-deploy-guide/tripleo-docs/wallaby/features/deployed_ceph.html#example-directly-setting-network-and-ms-bind-options

Reply:
indeed, trying now with @fultonj suggestion above

Reply:
I'm hoping `--single-host-defaults` is sufficient but if it's not we can use `--config initial-ceph.conf`

Reply:
it seems to be enough @fultonj thanks, the problem I'm facing now is that the compute node is not being deployed, something must be wrong in the parameters I'm passing to `openstack overcloud deploy`